code/validation_json_schemas.py

```diff
@@ -0,0 +1,190 @@
+# TODO validate type is in valid set of values for all below
+
+fv_schema = {
+    "type": "object",
+    "properties": {
+        "rate": {
+            "type": "number"
+        },
+        "nper": {
+            "type": "number"
+        },
+        "pmt": {
+            "type": "number"
+        },
+        "pv": {
+            "type": "number"
+        },
+        "type": {
+            "type": "number"
+        }
+    },
+    "required": ["rate", "nper", "pmt", "pv"]
```

Comment:
pv is optional. Should enforce the same constraints as Excel (see their help docs).